conda_environment.yml

```diff
@@ -3,7 +3,7 @@ channels:
 - defaults
 - conda-forge
 dependencies:
-- dask-core >2.0
+- dask-core >2.0,<2021.3.1
```

Comment:
It seems that Anaconda defaults is missing the `dask-core` 2021.3.1 packages and the latest package is picked up from conda-forge, which breaks dask...

Reply:
I have removed the pinning since the package is now available. It seems that it was a server issue.